### ISSUE TITLE
I want the ability to blacklist apps from connecting to the internet

### ISSUE THREAD

**Issue by StarTrekRocks73** [blokadaorg/blokada#189]
Hi Everyone,

### Expected behaviour

I want the ability to selectively block system apps, user apps, and the Android OS from connecting to the internet. There is a feature under Whitelist section that allows me to whitelist apps from being blocked by Blokada. I want a feature where a user can select and block a system app, user app, or the Android OS (ex. system.android.com) in the Blacklist section using a menu that is similar to the menu found under the Whitelist section.

### Actual behaviour

Currently there is no option to blacklist system apps, user apps, or the Android OS from connecting to the internet.

Sincerely,
StarTrekRocks73

**Comment by rpeter85** [comment]
Most probably a duplicate of #49 (https://github.com/blokadaorg/blokada/issues/49)

**Comment by StarTrekRocks73** [comment]
I just read issue #49 . Is it still not possible to implement this feature? I think that it would make this app more useful and help to gain more users. Plus, I really want the feature. :grin:
